addons/mass_mailing/migrations/12.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -65,10 +65,12 @@ NEW ir.ui.view: mass_mailing.mass_mailing_list_merge_view_form
 NEW ir.ui.view: mass_mailing.mass_mailing_mail_layout
 NEW ir.ui.view: mass_mailing.mass_mailing_mail_style
 NEW ir.ui.view: mass_mailing.mass_mailing_schedule_date_view_form
+NEW ir.ui.view: mass_mailing.view_mail_mass_mailing_list_form_simplified
+DEL ir.ui.view: mass_mailing.view_company_form_inherit_mass_mailing
+# NOTHING TO DO
+
 NEW ir.ui.view: mass_mailing.page_unsubscribe
 NEW ir.ui.view: mass_mailing.page_unsubscribed
 NEW ir.ui.view: mass_mailing.unsubscribe
 NEW ir.ui.view: mass_mailing.unsubscribed
-NEW ir.ui.view: mass_mailing.view_mail_mass_mailing_list_form_simplified
-DEL ir.ui.view: mass_mailing.view_company_form_inherit_mass_mailing
-# NOTHING TO DO
+# DONE: post-migration: moved from website_mass_mailing
```

Comment:
hehe, being a bit picky, it should be pre-migration, not post

Reply:
:laughing: done

Reply:
done where? :S

Reply:
Yikes, done now. Multitasking is not for me, definitely...